### PR TITLE
Docs: Document wal_sender_archiving_status_interval GUC

### PR DESCRIPTION
**Pull request by ShirishaRao**

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md [hunk 1]
```diff
@@ -3277,6 +3277,16 @@ The value of [wal\_sender\_timeout](#replication_timeout) controls the time that
...
 
+## <a id="wal_sender_archiving_status_interval"></a>wal\_sender\_archiving\_status\_interval
+
+When Greenplum Database segment mirroring and archiving is enabled, specifies the interval in milliseconds at which the walsender process on the primary segment sends archival status message to the walreceiver process on its mirror segment.
+
+A value of 0 deactivates this feature.
+
+|Value Range|Default|Set Classifications|
+|-----------|-------|-------------------|
+|0 - `INT_MAX`|10000 ms \(10 sec\)|master, system, reload|
```
Comment (kaknikhil) [inline gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:3288]: Do we need to set "superuser" for this guc ? @jimmyyih

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md [hunk 2]
```diff
@@ -3277,6 +3277,16 @@ The value of [wal\_sender\_timeout](#replication_timeout) controls the time that
 |-----------|-------|-------------------|
 |integer 0- INT\_MAX/1000|10 sec|master, system, reload, superuser|
 
+## <a id="wal_sender_archiving_status_interval"></a>wal\_sender\_archiving\_status\_interval
+
+When Greenplum Database segment mirroring and archiving is enabled, specifies the interval in milliseconds at which the walsender process on the primary segment sends archival status message to the walreceiver process on its mirror segment.
```
Comment (jimmyyih) [inline gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:3282]:
```suggestion
When Greenplum Database segment mirroring and archiving is enabled, specifies the interval in milliseconds at which the `walsender` process on the primary segment sends archival status messages to the `walreceiver` process of its corresponding mirror segment.
```

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md (hunk 1 above)
Comment (jimmyyih) [inline gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:3288]:
```suggestion
|0 - `INT_MAX`|10000 ms \(10 seconds\)|local, system, reload|
```

Comment (jimmyyih) [reply gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:3288]: It's not a session or superuser-hidden GUC so no.

**Review by jimmyyih — APPROVED** [review]

**Pull request merged**